config/vagrant/scripts/bootstrap-docker-ubuntu.sh

```diff
@@ -1,12 +1,9 @@
 #!/usr/bin/env bash
 
-# update docker config
-sudo bash -c 'cat <<- EOF > /etc/docker/daemon.json
-{
-  "ipv6": true,
-  "fixed-cidr-v6": "2001:db8:1::/64"
-}
-EOF'
+# update docker config to include ipv6 support
+jq '. +={"ipv6": true, "fixed-cidr-v6": "2001:db8:1::/64"}' < /etc/docker/daemon.json > /tmp/new-docker-daemon.json
```

Comment:
Yes, you're right on the "<", I'll remove it.

Regarding the "edit-in-place" tricks, I tested a version sending the content to an env var first, then echo'ing it to `/etc/docker/daemon.json` in just one line (otherwise the output redirection won't be sudoed), but didn't like it and finally discarded: harder to read/maintain. 
```
sudo bash -c 'content=$(jq ". +={\"ipv6\": true, \"fixed-cidr-v6\": \"2001:db8:1::/64\"}" /etc/docker/daemon.json) ; echo $content | jq > /etc/docker/daemon.json'
```
Same with sed/perl/super-clever-commands, whose syntax I'll forget ten minutes after I've uploaded the PR.